web/src/components/tasks/anytime-action/tax-clearance-certificate/AnytimeActionTaxClearanceCertificateElement.test.tsx

```diff
@@ -120,6 +129,115 @@ describe("<AnyTimeActionTaxClearanceCertificateReviewElement />", () => {
       fireEvent.click(screen.getByText(Config.taxClearanceCertificateStep3.editButtonText));
       expect(screen.getByTestId("eligibility-tab")).toBeInTheDocument();
     });
+
+    it("renders the first step as complete only when on the eligibility or review tab", () => {
+      renderComponent({});
+      expect(screen.getByTestId(`stepper-0`).dataset.state).toEqual("INCOMPLETE-ACTIVE");
+      fireEvent.click(screen.getByTestId("stepper-1"));
```

Comment:
Pebble - we are moving away from data-testid, here is an [RTL](https://testing-library.com/docs/queries/about/#priority) article. I recently added this [PR](https://github.com/newjersey/navigator.business.nj.gov/pull/10113) where I've removed data-testid from this file, it might help.